src/map/party.cpp

```diff
@@ -745,6 +752,18 @@ void CParty::ReloadParty()
                     PChar->pushPacket(new CPartyMemberUpdatePacket(PPartyMember, j, memberinfo.flags, PChar->getZone()));
                     //if (PPartyMember != PChar)
                     //    effects->AddMemberEffects(PChar);
+
+                    // Inject the party leader's trusts into the party list
+                    CCharEntity* leader = (CCharEntity*)GetLeader();
+                    if (leader->PTrusts.size() > 0)
```

Comment:
redundant check with the range based for loop below

Reply:
oh, and `leader` to `PLeader`

Reply:
Done. (both)